src/mrvi/_model.py

```diff
@@ -743,13 +771,122 @@ def get_aggregated_posterior(
             dist.Normal(qu_loc, qu_scale),
         )
 
+    def differential_abundance(
+        self,
+        adata: AnnData | None = None,
+        sample_cov_keys: list[str] | None = None,
+        sample_subset: list[str] | None = None,
+        batch_size: int = 128,
+    ) -> xr.Dataset:
+        """
+        Compute the differential abundance between samples.
+
+        Computes the logarithm of the ratio of the probabilities of each sample
+        conditioned on the estimated aggregate posterior distribution of each cell.
+
+        Parameters
+        ----------
+        adata
+            The data object to compute the differential abundance for.
+            If not given, the data object stored in the model is used.
+        sample_cov_keys
+            Keys for covariates (batch, etc.) that should also be taken into account
+            when computing the differential abundance.
```

Comment:
specify that these are expected to be discrete. We could also, alternatively, have an internal procedure than bins continuous covariates. What would you prefer?

Reply:
for now just expect discrete